pandas/tests/series/test_ufunc.py

```diff
@@ -439,3 +439,13 @@ def test_outer():
 
     with pytest.raises(NotImplementedError, match=tm.EMPTY_STRING_PATTERN):
         np.subtract.outer(s, o)
+
+
+def test_npmul():
+    # https://github.com/pandas-dev/pandas/issues/26650
+    data = np.random.randint(0, 10, 100, dtype="int64")
+    df = pd.DataFrame({"nums": data})
+    dfT = df.T
+
+    expected_result = pd.DataFrame(index=["nums"], data=[np.sum(np.square(data))])
```

Comment:
Could you create the result without relying on other functions?

Reply:
done